wagtail_content_import/parsers/tables.py

```diff
@@ -0,0 +1,13 @@
+class Table:
+
+    def __init__(self, cells=[]):
```

Comment:
We should probably remove the default here and make the caller pass in an empty list if they want to construct a table with no rows.

Reply:
That makes sense, will do.

Reply:
Also, we should avoid using mutable objects as defaults, since the same object is reused on every invocation. If it's accidentally modified in one table, it'll be modified in all the others as well.

Reply:
Ah oops, I'd forgotten that could happen! Thanks, will avoid in future

Reply:
Fixed now